extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -19,6 +19,7 @@
     "@theia/mini-browser": "next",
     "@theia/plugin-ext": "next",
     "axios": "0.19.0",
+    "@kubernetes/client-node": "^0.12.1",
```

Comment:
I think that I would expose kubernetes-api from a separate extension (like theia-plugin-remote extension that is exposing che-workspace-client to all extensions) and then re-expose it from plug-in-ext (so it's available from extension world and plug-ins world. You will argue that with inversify we can inject from anywhere but it's strange to have some extensions that would need to depends on plug-in-ext just to do k8s call)

I already extracted workspace-client from plug-in ext to cut the direct dependency so here we kinda doing again this dependency

TL;DR; I would move che-k8s-service.ts interface to a separate extension https://github.com/eclipse/che-theia/tree/master/extensions/eclipse-che-theia-remote-api/src/common and allow both client and server access

Reply:
updated